assets/js/base/components/cart-checkout/totals/footer-item/index.js

```diff
@@ -38,34 +39,36 @@ const TotalsFooterItem = ( { currency, values } ) => {
 	const parsedTaxValue = parseInt( totalTax, 10 );
 
 	return (
-		<TotalsItem
-			className="wc-block-components-totals-footer-item"
-			currency={ currency }
-			label={ label }
-			value={ parseInt( totalPrice, 10 ) }
-			description={
-				SHOW_TAXES &&
-				parsedTaxValue !== 0 && (
-					<p className="wc-block-components-totals-footer-item-tax">
-						{ createInterpolateElement(
-							__(
-								'Including <TaxAmount/> in taxes',
-								'woo-gutenberg-products-block'
-							),
-							{
-								TaxAmount: (
-									<FormattedMonetaryAmount
-										className="wc-block-components-totals-footer-item-tax-value"
-										currency={ currency }
-										value={ parsedTaxValue }
-									/>
+		<ExperimentalFooterTotalMeta>
```

Comment:
@nerrad you suggested rendering the `TotalsItem` in the Slot/Fill to dogfood our API. This isn't necessary, but we can do it if we want to.